app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+import os
+
+
+def move_file(command):
+    split_file = command.split(" ")
+    if "/" in split_file[-1]:
+        path = split_file[-1].split("/")
+        os.makedirs("/".join(path[:-1]))
```

Comment:
Notice that if you will try to create the directory already exists then it 
will lead to an error.